### PR TITLE
feat: Add Environment to auth flow

### PR DESCRIPTION
**Pull request by gregor**

### CODE REVIEW

File: app/script/util/Environment.js [hunk 1]
```diff
@@ -114,8 +114,8 @@ window.z.util = z.util || {};
   };
 
   const formatted_app_version = () => {
-    const version = app_version().split('-');
-    return `${version[0]}.${version[1]}.${version[2]}.${version[3]}${version[4]}`;
+    const [year, month, day, hour, minute] = app_version().split('-');
+    return `${year}.${month}.${day}.${hour}${minute}`;
```
Comment (bennycode) [inline app/script/util/Environment.js:118]: very good improvement! 👍

Comment (lipis) [reply app/script/util/Environment.js:118]: Solving magic numbers like boss... :)

That's why is nice to have this rule..

Comment (bennycode) [reply app/script/util/Environment.js:118]: I took this as an example: https://vote4code.com/5715683958587392/array-destructuring 💃

Comment (lipis) [reply app/script/util/Environment.js:118]: use camelCase :P `formattedAppVersion`

**Pull request merged**